app/config.py.example

```diff
@@ -63,6 +63,8 @@ email_conf = ConnectionConfig(
 JWT_KEY = "JWT_KEY_PLACEHOLDER"
 JWT_ALGORITHM = "HS256"
 JWT_MIN_EXP = 60 * 24 * 7
+CALENDAR_RESET_PASSWORD_PAGE = "calendar.pythonic.guru/reset-password"
```

Comment:
Prefer relative

Reply:
This link is being used in a sent email, and its purpose is to redirect the user back to the website